examples/starter/src/store.js

```diff
@@ -0,0 +1,7 @@
+/* eslint-disable import/prefer-default-export */
+import React from 'react';
+
+// Usage:
+//   <Provider store context={ApplicationStoreContext} />
+//   <ConnectedComponent context={ApplicationStoreContext} />
+export const ApplicationStoreContext = React.createContext(null);
```

Comment:
I guess this is now "clashing" with the create-mc-app (and hence CT docs)? I am getting a worried by us starting dependent changes in parallel without finishing the prior.

Reply:
I haven’t updated this branch after merging create-mc-app. Once I do, changes are properly reflected there as well